server/src/database/dummy-data/patient-general-therapy.js

```diff
@@ -0,0 +1,66 @@
+import { query } from '../connect';
+
+const createTherapy = async ({
+  patientId,
+  generalTherapyId,
+  therapyNumber,
+  notes,
+}) => {
+  const sql = `INSERT INTO patient_general_therapy(
+    patient_id,
+    global_therapy_id,
+    therapy_number,
```

Comment:
```suggestion
    patient_id,
   general_therapy_id,
    therapy_number,
```
Also, fix it in the dummy data.

Reply:
done